app/public/index.html

```diff
@@ -8,6 +8,7 @@
     <meta name="keywords" content="crypto, wallet, oauth, login, best, ethereum, torus">
     <meta name="author" content="Torus Labs">
     <link rel="shortcut icon" href="<%= BASE_URL %>favicon.png" />
+    <link href="https://file.myfontastic.com/upUhdYWuxL9ArmsPAbofL6/icons.css" rel="stylesheet"
```

Comment:
we are already using a lot of icons. please download the required icon and put it into the public folder and use it in the app